{{ cookiecutter.package_name }}/pyproject.toml

```diff
@@ -46,10 +46,30 @@ docs = [
   "packaging",
 ]
 
-{%- if cookiecutter.project_url %}
 [project.urls]
```

Comment:
Am I right in thinking that if *none* of these URLs are specified we would generate an invalid conifg?